sdk/python/packages/flet-core/src/flet_core/page.py

```diff
@@ -51,12 +52,20 @@
     ScrollMode,
     ThemeMode,
 )
-from flet_core.utils import deprecated
+from flet_core.utils import deprecated, classproperty
 from flet_core.utils.concurrency_utils import is_pyodide
 from flet_core.view import View
 
 logger = logging.getLogger(flet_core.__name__)
 
+_session_page = ContextVar("flet_session_page", default=None)
+
+
+class context:
```

Comment:
Shouldn't be `context` exported in __init__ to be available as `ft.context`?

Reply:
I'm not sure I understand you correctly. As I find it very convenient to refer to context via `ft`. I use `ft` for all controls in my code.

Reply:
I mean `context` class should be imported in flet-core's `__init__.py`, no?

Reply:
Oh, that's right. My mistake. Forgot to add it to this branch)
Give me five minutes and I'll fix it.

Reply:
Done